datalake-api/src/main/java/com/sequenceiq/sdx/api/endpoint/SdxEndpoint.java

```diff
@@ -127,7 +128,7 @@ SdxClusterResponse create(@ValidStackNameFormat @ValidStackNameLength @PathParam
     @Path("{datalakeName}/refresh")
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = "Restart and reload all configurations of the data hub by name", produces = "applicaton/json", nickname = "refreshDatahubs")
-    SdxClusterResponse refreshDataHubs(@PathParam("datalakeName") String name, @QueryParam("datahubName") String datahubName);
+    SdxRefreshDatahubResponse refreshDataHubs(@PathParam("datalakeName") String name, @QueryParam("datahubName") String datahubName);
```

Comment:
Changing the response return type will likely make thunderhead clients unhappy (or whoever else directly calling this endpoint), so please be sure to update the datalake-client there as well if needed. :smile:

Reply:
Kibana checked. There is no usage of this API. So this change will not break the API.
Used query: method: SdxController#refreshDataHubs interval: last 30 days, which is enough because the API creation was three weeks ago.

And I also have a ticket for the thunderhead part. So I'll change it after that merge.